plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-product-images.php

```diff
@@ -79,6 +79,7 @@ public static function output( $post ) {
 		</div>
 		<p class="add_product_images hide-if-no-js">
 			<a href="#" data-choose="<?php esc_attr_e( 'Add images to product gallery', 'woocommerce' ); ?>" data-update="<?php esc_attr_e( 'Add to gallery', 'woocommerce' ); ?>" data-delete="<?php esc_attr_e( 'Delete image', 'woocommerce' ); ?>" data-text="<?php esc_attr_e( 'Delete', 'woocommerce' ); ?>"><?php esc_html_e( 'Add product gallery images', 'woocommerce' ); ?></a>
+			<?php echo wc_help_tip( __('For best results, upload JPEG or PNG files that are 1000 by 1000 pixels or larger. Maximum upload file size: 2GB.', 'woocommerce' ) ) ?>
```

Comment:
Can you use the `i18n_product_image_tip` in `class-wc-admin-assets.php` above>

Reply:
Ha, nevermind... 😄 I failed to notice that the `i18n_product_image_tip` was actually getting put into a JS script. Hmm... I guess the only ways to avoid this duplication would be to...

1. move this out to a separate global that could be references by both `class-wc-admin-assets.php` and this php file, or
2. move the creation of this tooltip to JS as the other one is being done (though there is no other reason for it to be done in JS)

I don't hate option (2)... it brings a certain consistency to how they are both handled.